lib/cf/applications/src/app-event-converter.js

```diff
@@ -12,7 +12,7 @@ const isPurgeEvent = (event) => {
 
 const createDefaultMeasures = (event) => {
   return {
-    previousRunningInstances: event.entity.previous_instance_count,
+    previousRunningInstances: event.entity.previous_instance_count || event.entity.instance_count,
     previousInstanceMemory: event.entity.previous_memory_in_mb_per_instance * 1048576,
```

Comment:
We should protect previous memory as well

Reply:
Since we haven't see such an issue before we decided to default only previous_instance_count to instance_count and leave the memory without defaulting.

Reply:
Seems legit. It would be interesting to see if we **can** have event without change in instance or memory.